docs/basics/integrations/tosca.md

```diff
@@ -1,11 +1,75 @@
 ---
 id: tosca
 title: Tosca Integration
-sidebar_label: Tosca 
+sidebar_label: Tosca
 description: Create, execute, and maintain test easily with the industry's leading model-based test platform
-keywords:
-- automated-testing
-- mobile-testing
-- model-based testing
-- how-to
 ---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Tricentis Tosca is an industry-leading test creation tool, allowing for rich test case automation against web, mobile, and client-server applications. When you need to execute a lot of Tosca tests in parallel, look to Sauce Labs to help you scale.
+
+## Overview
+
+Tosca scans your app to gather data about the various pages and elements. From the model Tosca creates, you will piece together sequences of actions, allowing you to script the steps your test will execute. With the Sauce Labs credentials in place, you then choose the platforms you want to execute on. See the [Platform Configurator](https://saucelabs.com/platform/platform-configurator) for more information.
+
+## What You’ll Need
+
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+* Your Sauce Labs [Username](https://app.saucelabs.com/user-settings).
+* You’ll also need to download and install Tosca.
+
+## Scanning an App
+
+1. In the **Tosca Commander**, on the **Modules** tab, click **Scan** and then click **Mobile**.
+
+<img src={useBaseUrl('img/integrations/tosca-modules.png')} alt="Tosca Commander Modules tab" width="600"/>
```

Comment:
The images for each step should be indented -- the current alignment is off.

<img width="716" alt="Screen Shot 2022-11-25 at 12 56 51 PM" src="https://user-images.githubusercontent.com/15352964/204036285-edb7c4a8-9041-4141-827e-b4994ab63bf6.png">